app/waf/src/controllers/adc.controller.ts

```diff
@@ -451,15 +449,13 @@ export class AdcController extends BaseController {
   //     "id": "11111111-2222-3333-4444-555555555555"
   //   }
   // }
-  @post(prefix + '/adcs/{adcId}/action', {
+  @post(prefix + '/adcs/{adcId}/setup', {
     responses: {
       '200': Schema.response(ActionsResponse, 'Adc Id for the actions.'),
```

Comment:
Not quite sure if we will still need this response body. Usually 204 with empty body will be OK for an asyncronous API. We can decide that later.